backend/layers/common/entities.py

```diff
@@ -81,6 +81,7 @@ class DatasetArtifactType(str, Enum):
     RDS = "rds"
     CXG = "cxg"
     ATAC_FRAGMENT = "fragment_tsv"
+    ATAC_INDEX = "fragment_index"
```

Comment:
is there a reason the names don't match? Same for ATAC_FRAGMENT
```suggestion
    ATAC_INDEX = "atac_index"
```

Reply:
This is what it says to do in the spec [here](https://docs.google.com/document/d/1tNoh1J6bM8NByJRIqiiT3P0BfdyBVjjZ9fjrEJ0l8Y0/edit?tab=t.0#heading=h.pceclqpxaox1)